agenda/agenda_3.md

```diff
@@ -5,4 +5,7 @@
 
 **Agenda**
 
-TBD
\ No newline at end of file
+- [SIMD-16 - Application account write fees](https://github.com/solana-foundation/solana-improvement-documents/pull/16) 
+  [PR](https://github.com/solana-labs/solana/pull/30137)
+
+- [QUIC Bidirectional Replies](https://github.com/solana-labs/solana/pull/30161) [PR](https://github.com/solana-labs/solana/pull/29954)
```

Comment:
QUIC Bidirectional is client specific and may not be applicable to the entire core community audience. Let's discuss this one in discord

Reply:
Ok removed the QUIC Direction replies